cov.py

```diff
@@ -68,22 +75,33 @@ def collect_cov(log_dir, out, iss, testlist, batch_size, lsf_cmd, steps, \
     for i in range(len(log_list)):
       log = log_list[i]
       csv = csv_list[i]
-      logging.info("Process %0s log[%0d/%0d] : %s" % (iss, i+1, len(log_list), log))
-      if iss == "spike":
-        process_spike_sim_log(log, csv, 1)
-      elif iss == "ovpsim":
-        process_ovpsim_sim_log(log, csv, 1, stop_on_first_error)
+      # If a core target is defined, prioritize over ISS
+      if core:
+        logging.info("Process %0s log[%0d/%0d] : %s" % (core, i+1, len(log_list), log))
+        if core == "ibex":
+          ibex_log_cmd = ("python3 %s/ibex_log_to_trace_csv.py --log %s --csv %s --full_trace 1" %
```

Comment:
Can we have a way to extend core specific behavior in ibex repo? I think it's better to not include core specific code in riscv-dv.

Reply:
Something that might be easier actually, is to default enable the full trace parsing of the ibex logs, so that calling the parsing scripts again is not necessary, what do you think?

Reply:
yes, default enable full_trace sounds good to me.

Reply:
Done, see https://github.com/lowRISC/ibex/pull/453